action/init.go

```diff
@@ -5,9 +5,14 @@ import (
 
 	"github.com/fatih/color"
 	"github.com/justwatchcom/gopass/gpg"
+	"github.com/mattn/go-colorable"
```

Comment:
I wonder why we'd need another color library along the existing one. If you believe your lib provides a benefit over the existing one feel free to open an separate PR to completely replace the old one, but for this PR I'd prefer if we could stick to the existing one.